pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter_test.go

```diff
@@ -405,6 +405,22 @@ var _ = Describe("Validating VirtualMachineClone Admitter", func() {
 		)
 	})
 
+	Context("newMacAddresses", func() {
```

Comment:
nit: I think this fits a `DescribeTable`.

Reply:
yes, follow

Reply:
done